src/components/address/AddressLookup/index.js

```diff
@@ -135,7 +142,7 @@ module.exports = React.createClass({
       address: null,
       custom: null,
       focusOnMount: true
-    }, this.output);
+    }, callback.apply(this, arguments));
```

Comment:
This seems odd. If you need access to the component object, why not just put the callback function on the component?